features/vendor_provider_download.feature

```diff
@@ -20,7 +20,7 @@ Scenario:
         
     Then I am on the user login page
     Then I am signed in as a vendor "Github" and user ID "12345" with "facebook"
-    And I have updated the vendor profile
+    And I have updated the vendor home
```

Comment:
I'm hoping this is correct, although I'm not entirely sure of the difference between the 'profile' and 'home' pages for vendor.